### DISCUSSION TITLE
Workspace emoji picker should be searchable

### DISCUSSION THREAD

**Discussion by akashbagchi** [zen-browser/desktop#2617] [Ideas]
The Emoji picker for workspace creation/editing should have a search function to quickly filter based on keyword. As it currently stands, the limited selection of emojis + lack of search function make it so I need to scroll through the entire collection while squinting and trying to figure out if there's any option that even remotely matches the icon I'm looking for.

<img width="1904" alt="Screenshot 2024-11-04 at 6 11 24 AM" src="https://github.com/user-attachments/assets/38bf6494-fb18-4941-8306-634e80e63d80">

^ Very hard to discern the emojis on a 13" display, but I'm stubborn when it comes to having a very zoomed-out view of things.

#### Recommended/Requested features:

- Search bar to filter emojis by keyword
- Tooltip that tells me the name of which emoji I'm currently hovering over

**Comment by jdvlpr** [comment]
I agree!

**Comment by amekhalfia04** [comment]
I agree. And the emojis should also be a bit bigger. I don't know if its my display settings or font size, but they look far too small and I find myself breaking my neck to look closer at each one.

**Reply by BoxedFruits** [reply]
I think it could be nice to make the non-focused workspace emoji's not as dark as well. They're almost indiscernible when greyed out

**Comment by itaakesama** [comment]
It is really needed

**Comment by 12th-devs** [comment]
https://github.com/zen-browser/desktop/pull/3963

**Comment by XanderLeaDaren** [comment]
+ Why is there only a subset of the whole Emoji icons? I’m missing a few to precisely indicate some workspaces, here!
